charts/telepresence/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Feature: Add AgentInjectorWebhook yaml files, newly introduced in 2.3.1.
 - Feature: Include service account + ClusterRole + ClusterRoleBinding for the traffic-manager, since it now communicates directly with the cluster.
+- Feature: add `priorityClassName` value.
```

Comment:
I don't think we're using this `CHANGELOG.md` file, but let it be here for now. We can move this line up to the root after it's been merged and we have a new release pending.